code/game/machinery/medical_kiosk.dm

```diff
@@ -236,7 +236,13 @@
 			chemical_list += list(list("name" = R.name, "volume" = round(R.volume, 0.01)))
 			if(R.overdosed)
 				overdose_list += list(list("name" = R.name))
-
+	var/obj/item/organ/stomach/belly = altPatient.getorganslot(ORGAN_SLOT_STOMACH)
+	if(belly?.reagents.reagent_list.len) //include the stomach contents
```

Comment:
Does the `reagents` datum always exist on the stomach? Does `reagent_list` as well?

Reply:
This was a check for belly, and Ien and thought this was a shorter form of `(belly && belly.reagents.reagent_list.len)`

Reply:
I do not know of a way that an atom can loss it's reagent holder.